content-repo/extra-docs/releases/20.11.1.md

```diff
@@ -0,0 +1,1116 @@
+# Cortex XSOAR Content Release Notes for version 20.11.1 (196572)
+##### Published on 22 November 2020
+
+### Breaking Changes
+Several packs include breaking changes.
+
+- [CrowdStrike Malquery Pack v1.0.1](#crowdstrike-malquer-pack-v101)
+- [PAN-OS Pack v1.6.6](#pan-os-pack-v166)
+- [PassiveTotal Pack v2.0.5](#passivetotal-pack-v205-partner-supported)
+
+### New: Asset Pack v1.0.0
```

Comment:
Let's add description what this pack is. It is not clear. Add the following:

Base pack with incident fields for any packs requiring asset fields.